packages/framework-integration-tests/integration/end-to-end/cart.integration.ts

```diff
@@ -435,4 +435,35 @@ describe('Cart end-to-end tests', () => {
       })
     })
   })
+
+  describe('Scheduled commands', () => {
+    it('scheduled command ran and created a product', async () => {
+      const scheduledProductSku = 'scheduled-product-created'
+
+      // Check that scheduled command created the new product
+      const products = await waitForIt(
+        () => {
+          return client.query({
+            query: gql`
+              query {
+                ProductReadModels {
+                  id
+                  sku
+                  displayName
+                  description
+                  price
+                  availability
+                  deleted
+                }
+              }
+            `,
+          })
+        },
+        (result) => result?.data?.ProductReadModels?.some((product: any) => product.sku === scheduledProductSku)
+      )
+
+      const product = products.data.ProductReadModels.find((product: any) => product.sku === scheduledProductSku)
+      expect(product).not.to.be.null
```

Comment:
Great test!

Reply:
Thanks, @javiertoledo! You can find some notes about it in the [README](https://github.com/boostercloud/booster/blob/a10f305a7502a7cfd7ac367db5cb31a420f063b9/docs/README.md#scheduling-a-command)